vsphere/data_source_vsphere_network.go

```diff
@@ -36,6 +36,12 @@ func dataSourceVSphereNetwork() *schema.Resource {
 				Description: "Id of the distributed virtual switch of which the port group is a part of",
 				Optional:    true,
 			},
+			"additional_filters": {
+				Type:        schema.TypeMap,
+				Description: "Additional filters to uniquely identify the network",
```

Comment:
Should mention supported filters?

Reply:
i am pulling this out and calling it network_type